src/games/gamepanelwidget.py

```
@@ -0,0 +1,338 @@
+        if pixmap is None:
+            self.ui.mapLabel.setPixmap(util.THEME.pixmap("games/unknown_map.png").scaled(256, 256))
+        else:
+            self.ui.mapLabel.setPixmap(pixmap.scaled(256, 256))
+        self.ui.getMapButton.setEnabled(not fa.maps.isMapAvailable(self.game.mapname))
+
+    def refresh_ui(self) -> None:
+        if self.game is None:
+            return
+        self.on_game_updated(self.game, self.game)
+
+    def on_game_updated(self, new: Game, old: Game) -> None:
```

Ruff (0.14.5)

Unused method argument: `old`

(ARG002)